### PR TITLE
Add/unit test for color change propagation

### PR DESCRIPTION
**Pull request by sasensi**
#### Related issues

- Relates to #1672

### Checklist

- [x] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the JSHint rules (`npm run jshint` passes)

### CODE REVIEW

**Comment by sasensi** [comment]
@lehni, the PR worked this time, it now includes the requested changes.

**Comment by lehni** [comment]
Thank you @sasensi! But you forgot this one: 
> Could this state `'canvas content should change after a change of item.fillColor'` instead?

File: test/tests/Style.js [hunk 1]
```diff
@@ -192,3 +192,27 @@ test('setting Group#fillColor and #strokeColor 2', function() {
...
+        radius: 70,
+        fillColor: 'red'
+    });
+    var imageDataBefore = getDataURL();
+
+    // Change style property and check that change was detected.
+    item.fillColor.hue += 100;
+    var imageDataAfter = getDataURL();
+
+    // Canvas content should change after a change of item.fillColor.
```
Comment (sasensi) [inline test/tests/Style.js]: @lehni the comment is here, did you want me to put it somewhere else ?

Comment (lehni) [reply test/tests/Style.js]: Oh I didn't see it! I meant the string in the `equals()` call, as that's clearer wha the intention was, should it ever fail again.

Comment (sasensi) [reply test/tests/Style.js]: Oh ok, I will change it then

**Comment by sasensi** [comment]
@lehni, ok, I added the test error message.

**Pull request merged**